core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -21,7 +21,8 @@ Behaviour.specify("#filter-box", '_table', 0, function(e) {
             }
             var makeVisible = true;
 
-            var content = items[i].innerHTML.toLowerCase();
+            var pluginId = items[i].getAttribute('data-plugin-id');
+            var content = (items[i].querySelector('.details').innerText + pluginId).toLowerCase();
```

Comment:
```suggestion
            var content = (items[i].querySelector('.details').innerText + " " + pluginId).toLowerCase();
```

Currently, finding Structs Plugin when filtering for `objects.str` seems weird. Who knows what weird accidental matches are possible this way?

Untested.

Reply:
I was looking for `putti` (typoed "PuTTY", don't judge) and found Timestamper 😄 

> ![Screenshot](https://user-images.githubusercontent.com/1831569/139958195-f5b2f947-a99c-4c69-8f2f-b20e288f48be.png)